src/main/java/org/sonarsource/sonarlint/ls/telemetry/SonarLintTelemetry.java

```diff
@@ -87,6 +88,10 @@ public void helpAndFeedbackLinkClicked(String itemId) {
     actIfEnabled(telemetryRpcService -> telemetryRpcService.helpAndFeedbackLinkClicked(new HelpAndFeedbackClickedParams(itemId)));
   }
 
+  public void toolCalled(String toolName, boolean success) {
+    actIfEnabled(telemetryRpcService -> telemetryRpcService.toolCalled(new ToolCalledParams(toolName, success)));
```

Comment:
Not related to your change, but as long as the telemetry backend capability is not enabled at startup, you shouldn't have to verify for each call to "actIfEnabled"

Reply:
I guess the check is meant for tests, so that we can test both scenarios (enabled/disabled)?